geoutils/georaster/raster.py

```diff
@@ -1002,6 +1002,22 @@ def copy(self: RasterType, new_array: np.ndarray | None = None) -> RasterType:
 
         return cp
 
+    def get_nanarray(self, return_mask: bool = False) -> np.ndarray | tuple[np.ndarray, np.ndarray]:
```

Comment:
SO that mypy does not annoy us in theh future, it would be good to write the overload now.